documentation/src/main/asciidoc/topics/con_file_based_cache_stores.adoc

```diff
@@ -1,72 +1,36 @@
 [id='file-stores_{context}']
 = File-based cache stores
 
-File-based cache stores provide persistent storage on the local host filesystem where {brandname} is running.
-For clustered caches, file-based cache stores are unique to each {brandname} node.
+File-based cache stores provide persistent storage on the local host filesystem where {brandname} is running. 
 
-[WARNING]
-====
-Never use filesystem-based cache stores on shared file systems, such as an NFS or Samba share, because they do not provide file locking capabilities and data corruption can occur.
-
-Additionally if you attempt to use transactional caches with shared file systems, unrecoverable failures can happen when writing to files during the commit phase.
-====
-
-[discrete]
-== Soft-Index File Stores
-
-`SoftIndexFileStore` is the default implementation for file-based cache stores and stores data in a set of append-only files.
-
-When append-only files:
-
-* Reach their maximum size, {brandname} creates a new file and starts writing to it.
-* Reach the compaction threshold of less than 50% usage, {brandname} overwrites the entries to a new file and then deletes the old file.
-
-.B+ trees
-
-To improve performance, append-only files in a `SoftIndexFileStore` are indexed using a **B+ Tree** that can be stored both on disk and in memory.
-The in-memory index uses Java soft references to ensure it can be rebuilt if removed by Garbage Collection (GC) then requested again.
+The following types of file-based cache stores exist in {brandname}:
+* Shared, where cache entries persist after a cluster restart operation. Examples include SQL, JDBC, and so on. 
+* Non-shared, where cache entries do not persist after a cluster restart operation. Examples include RocksDB, single file, and so on.
```

Comment:
This isn't correct. The entries in both shared and non-shared WILL persist after a cluster restart, however it's not safe to use non-shared stores in a scenario where individual nodes are restarted as the store may contain stale entries for a given key.

Reply:
Thanks, @ryanemerson .

How about:

> Non-shared, where cache entries persist after a cluster restart operation, but in certain situations data loss might occur where individual nodes are restarted and the non-shared store contains stale key entries.

Reply:
Might I ask where the stale keys are stored?

Reply:
The key/values will be shared in the configured file-store, so on the user's disk.

I would hold off on this section until we make a decision on https://github.com/infinispan/infinispan/pull/10360#discussion_r995817310